library/objective-c/EnvoyBridgeUtility.h

```diff
@@ -100,6 +100,23 @@ static inline NSString *to_ios_string(envoy_data data) {
   return platformString;
 }
 
+static inline EnvoyEvent *to_ios_map(envoy_map map) {
+  NSMutableDictionary *newMap = [NSMutableDictionary new];
```

Comment:
The last time I looked at bytecode, I think `alloc`/`init` was still marginally preferable, but obviously this isn't a terrific concern.

Reply:
we use `new` in multiple places in this file already - for consistency sake will keep `new` here. This is something that will be hard to ensure unless we lint against the usage of `new` (if such linters exist)

Reply:
Fair enough.